tests/components/homewizard/test_switch.py

```diff
@@ -429,7 +429,7 @@ async def test_switch_handles_requesterror(
 async def test_switch_handles_disablederror(
     hass, mock_config_entry_data, mock_config_entry
 ):
-    """Test entity raises HomeAssistantError when Disabled was raised."""
+    """Test entity pytest.raises HomeAssistantError when Disabled was raised."""
```

Comment:
There are some more of these.
```suggestion
    """Test entity raises HomeAssistantError when Disabled was raised."""
```

Reply:
Search & replace at its best. Sorry about that one 🙏